packages/core/src/components/tooltip/tooltip.scss

```diff
@@ -0,0 +1,79 @@
+.atom-tooltip {
+  --arrow-width: 10px;
+  --arrow-height: 8px;
+
+  background: var(--color-neutral-light-1);
+  border-radius: var(--border-radius);
+  color: var(--color-neutral-white);
+  font: var(--text-body-small);
+  letter-spacing: var(--text-body-small-letter);
+  padding: var(--spacing-xsmall);
+  visibility: hidden;
+
+  &__arrow {
+    background: inherit;
+    height: var(--arrow-height);
+    position: absolute;
+    visibility: hidden;
+    width: var(--arrow-width);
+
+    &::before {
+      background: inherit;
+      content: '';
+      height: var(--arrow-height);
+      position: absolute;
+      transform: rotate(45deg);
+      width: var(--arrow-width);
+    }
+  }
+
+  &.open {
+    visibility: visible;
+  }
+
+  &.open > &__arrow {
+    &::before {
+      visibility: visible;
+    }
+  }
+
+  &[data-popper-placement^='top'] > &__arrow {
+    bottom: -3px;
+  }
+
+  &[data-popper-placement^='bottom'] > &__arrow {
+    top: -3px;
+  }
+
+  &[data-popper-placement^='left'] > &__arrow {
+    right: -3px;
+  }
+
+  &[data-popper-placement^='right'] > &__arrow {
+    left: -3px;
```

Comment:
(suggestion) maybe we could transform `-3px` in a variable?

Reply:
I started thinking more with SCSS, and choose to use a loop to perform this section. done in [99079d1](https://github.com/juntossomosmais/atomium/pull/367/commits/99079d1bec5252d85cacc605b49d0ca4727d17ee)